JavaIntegration/src/main/java/org/openzen/zencode/java/module/JavaNativeModule.java

```diff
@@ -79,9 +74,14 @@ public JavaNativeModule(
 	}
 
 	public SemanticModule toSemantic() {
-		return new SemanticModule(
+		SemanticModule[] semanticModules = dependencies.stream()
```

Comment:
❓ Question: Is this the proper way to handle dependencies to other JavaNativeModules?